xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/DXVAHD.cpp

```diff
@@ -324,16 +318,39 @@ bool CProcessorHD::InitProcessor()
         DXGI_COLOR_SPACE_RGB_FULL_G22_NONE_P709, &supported);
     const bool topLeft = SUCCEEDED(hr) && static_cast<bool>(supported);
 
+    // Check if driver prefers HDR10 LEFT input color space for HDR10 passthrough
+    const DXGI_COLOR_SPACE_TYPE outputCS = DX::Windowing()->UseLimitedColor()
+                                               ? DXGI_COLOR_SPACE_RGB_STUDIO_G2084_NONE_P2020
+                                               : DXGI_COLOR_SPACE_RGB_FULL_G2084_NONE_P2020;
+
+    hr = m_pEnumerator1->CheckVideoProcessorFormatConversion(
+        DXGI_FORMAT_P010, DXGI_COLOR_SPACE_YCBCR_STUDIO_G2084_LEFT_P2020, format, outputCS,
+        &supported);
+    const bool HDRleft = SUCCEEDED(hr) && static_cast<bool>(supported);
+
+    hr = m_pEnumerator1->CheckVideoProcessorFormatConversion(
+        DXGI_FORMAT_P010, DXGI_COLOR_SPACE_YCBCR_STUDIO_G2084_TOPLEFT_P2020, format, outputCS,
+        &supported);
+    const bool HDRtopLeft = SUCCEEDED(hr) && static_cast<bool>(supported);
+
     m_BT2020TopLeft = !left && topLeft;
```

Comment:
Should prefer topleft as well, this mode is used for HDR source, most likely a topleft uhd bd.
AMD supports topleft and left equally.
Please also log which one will be used when both are available,

Reply:
Done, only is logged when both are available.